core/drivers/regulator/regulator.c

```diff
@@ -258,7 +252,7 @@ void regulator_print_state(const char *message __maybe_unused)
 	DMSG("name     use\ten\tuV\tmin\tmax\tsupply");
 
 	SLIST_FOREACH(regulator, &regulator_device_list, link)
-		DMSG("%8s %u\t%d\t%d\t%d\t%d\t%#x\t%s\n",
+		DMSG("%8s %u\t%d\t%d\t%d\t%d\t%s\n",
```

Comment:
This should have been caught by CI (mismatch in number of arguments). Why not?

Reply:
No board currently enable `CFG_DRIVERS_REGULATOR`. I did enable the switch only i the [last commit](https://github.com/OP-TEE/optee_os/pull/6301/commits/12455cde3bd8522a0e23b27f5d5270dd4fc9b58b#diff-6301a2703b709daa4d84ead1c30bf67cdaf088abed68753a8f9c740f1e58b54aR540-R544) of my series in https://github.com/OP-TEE/optee_os/pull/6301. 
I'll append a commit to enable the switch on plat-stm32mp1 so that we at least build a platform with the framework.

Reply:
Yes, it is a good idea to add some CI (at least building) whenever new code is added.